content/articles/2019-02-stringr-1-4-0.Rmarkdown

```diff
@@ -0,0 +1,69 @@
+---
+title: stringr 1.4.0
+author: Mara Averick
+date: '2019-02-21'
+slug: stringr-1-4-0
+description: >
+    stringr 1.4.0 is now on CRAN!
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/o-d37kiKqqc
+  author: Maranda Vandergriff
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
+We are happy to announce that [stringr](http://stringr.tidyverse.org/) 1.4.0
+is now on CRAN. stringr provides a cohesive set of functions designed to make
+working with strings as easy as possible. For a full list of changes, please see the [release notes](https://stringr.tidyverse.org/news/index.html#stringr-1-4-0).
+
+You can install the released version from CRAN:
+
+```{r eval = FALSE}
+install.packages("stringr")
+```
+
+```{r}
+library(stringr)
+```
+
+
+## New functions
+
+Thanks to the hard work of [John Harmon](https://github.com/jonthegeek) at Tidyverse Developer Day, stringr has three new functions.
+
+[`str_starts()`](https://stringr.tidyverse.org/reference/str_starts.html) and [`str_ends()`](https://stringr.tidyverse.org/reference/str_starts.html) detect the presence or absence of patterns at the beginning or end of strings.
+ 
+```{r str-starts}
+fruit <- c("apple", "banana", "pear", "pineapple")
+str_starts(fruit, "p")
+str_starts(fruit, "p", negate = TRUE)
+str_ends(fruit, "e")
+str_ends(fruit, "e", negate = TRUE)
+```
+ 
+The new [`str_to_sentence()`](https://stringr.tidyverse.org/reference/case.html) function capitalizes strings with sentence case, like so:
+
+```{r str-sentence}
+str_to_sentence("the quick brown dog")
+```
+
+## Minor updates and fixes
+
+[`str_subset()`](https://stringr.tidyverse.org/reference/str_subset.html), [`str_detect()`](https://stringr.tidyverse.org/reference/str_detect.html), and [`str_which()`](https://stringr.tidyverse.org/reference/str_subset.html) now have the `negate` argument, which is used to find the elements that do _not_ match a pattern (as seen above in the `str_starts()` and `str_ends()` examples).  
+
+[`str_interp()`](https://stringr.tidyverse.org/reference/str_interp.html) now renders lists consistently independent of the presence of additional placeholders.  
```

Comment:
Drop this since I don't want to advertise `str_interp()` (`str_glue()` is the preferred alternative). Then maybe change heading to `negate` support?

Reply:
I did "Support for `negate`", just so we don't start a header with code.